autoload/limelight.vim

```diff
@@ -137,8 +137,12 @@ endfunction
 function! s:dim(coeff)
   let synid = synIDtrans(hlID('Normal'))
   let fg = synIDattr(synid, 'fg#')
-  let bg = synIDattr(synid, 'bg#')
-
+  if synIDattr(synid, 'bg#') == 'none'
+    let bg = 0
+  else
+    let bg = synIDattr(synid, 'bg#')
+  endif
+  
```

Comment:
We can remove redundant `synIDattr` calls.

```vim
let bg = synIDattr(synid, 'bg#')
let bg = bg == 'none' ? 0 : bg
```

Reply:
Sure! That simplifies it a lot.
Committed the changes.